packages/timeline-state-resolver/src/integrations/abstract/__tests__/abstract.spec.ts

```diff
@@ -1,185 +1,172 @@
-import { Mappings, DeviceType, Mapping, SomeMappingAbstract } from 'timeline-state-resolver-types'
-import { Conductor } from '../../../conductor'
-import { AbstractDevice } from '..'
+/* eslint-disable jest/expect-expect */
+import { TSRTimelineContent, TimelineContentAbstractAny, Timeline, DeviceType } from 'timeline-state-resolver-types'
+import { AbstractCommandWithContext, AbstractDevice, AbstractDeviceState } from '..'
 import { StatusCode } from '../../../devices/device'
 import { MockTime } from '../../../__tests__/mockTime'
-import { ThreadedClass } from 'threadedclass'
-import { getMockCall } from '../../../__tests__/lib'
+import { ResolvedTimelineObjectInstance } from 'timeline-state-resolver-types/dist/superfly-timeline'
+
+async function getInitialisedDevice() {
+	const dev = new AbstractDevice()
+	await dev.init({})
+
+	return dev
+}
 
 describe('Abstract device', () => {
 	const mockTime = new MockTime()
 	beforeEach(() => {
 		mockTime.init()
 	})
-	test('Abstract device methods', async () => {
-		const commandReceiver0: any = jest.fn(async () => {
-			return Promise.resolve()
-		})
-		const myLayerMapping0: Mapping<SomeMappingAbstract> = {
-			device: DeviceType.ABSTRACT,
-			deviceId: 'myAbstract',
-			options: {},
-		}
-		const myLayerMapping: Mappings = {
-			myLayer0: myLayerMapping0,
-		}
 
-		const myConductor = new Conductor({
-			multiThreadedResolver: false,
-			getCurrentTime: mockTime.getCurrentTime,
-		})
-		await myConductor.init()
-		await myConductor.addDevice('myAbstract', {
-			type: DeviceType.ABSTRACT,
-			options: {},
-			commandReceiver: commandReceiver0,
-		})
-		await mockTime.advanceTimeToTicks(10100)
+	test('Device setup', async () => {
```

Comment:
Perhaps this could have another test describing no commands should be sent, but it's fine without imo.